src/components/FSHControls.js

```diff
@@ -98,6 +98,16 @@ export default function FSHControls(props) {
   const [currentExample, setCurrentExample] = useState('');
   const [currentExampleName, setCurrentExampleName] = useState('');
 
+  if (!canonical && props.config?.canonical) {
+    setCanonical(props.config?.canonical);
```

Comment:
Is the `?.` necessary here if it has made it into the if statement?

Reply:
Fixed in https://github.com/FSHSchool/FSHOnline/pull/65/commits/56b12ac7d49d2f74b37c0ef479fb0d76f83b18e8